app/models/ability.rb

```diff
@@ -197,14 +190,13 @@ def signed_in_with_cfp_role(user)
 
     # Abilities for Role (Conference resource)
     can :index, Role
-
-    can :manage, Role do |role|
+    can :show, Role do |role|
       role.resource_type == 'Conference' && (conf_ids_for_cfp.include? role.resource_id)
     end
 
-    # Can add or remove users from role, when user has that same role for the conference
+    # Can add or remove users from role and edit role description, when user has that same role for the conference
     # Eg. If you are member of the CfP team, you can add more CfP team members (add users to the role 'CfP')
-    can :toggle_user, Role do |role|
+    can [:toggle_user, :update, :edit], Role do |role|
       role.resource_type == 'Conference' &&
       (Conference.with_role(role.name.parameterize.underscore.to_sym, user).pluck(:id).include? role.resource_id)
```

Comment:
`parameterize.underscore`  are they really required?
None of the 4 role names have any special characters in the.